server/server/data/src/main/java/com/alipay/sofa/registry/server/data/bootstrap/DataServerBeanConfiguration.java

```diff
@@ -363,12 +370,11 @@ public SnapshotBackUpNotifier snapshotBackUpNotifier() {
         }
 
         @Bean(name = "dataChangeNotifiers")
-        public List<IDataChangeNotifier> dataChangeNotifiers() {
+        public List<IDataChangeNotifier> dataChangeNotifiers(DataServerConfig dataServerBootstrapConfig) {
             List<IDataChangeNotifier> list = new ArrayList<>();
             list.add(sessionServerNotifier());
             list.add(tempPublisherNotifier());
             list.add(backUpNotifier());
-            list.add(snapshotBackUpNotifier());
             return list;
```

Comment:
why this change here?  `dataServerBootstrapConfig` param is unuseful and why `snapshotBackUpNotifier()` removed?